### PR TITLE
Modernizing php examples

### PR DESCRIPTION
**Pull request by SadMonk**
Effectively I just quickly fixed some php 7.0 code style warnings my IDE threw at me after looking at the example files.
The code autoformat I did afterwards should have employed PSR-12 rules.

### CODE REVIEW

File: php/examples/users.php [hunk 1]
```diff
@@ -20,58 +20,52 @@
 /**
  * @param array $installedUsers
  *
```
Comment (jgsf-web) [inline php/examples/users.php:22]: Why is this int to begin with? The PBX has login IDS as strings, e.g. 0001

Comment (SadMonk) [reply php/examples/users.php:22]: interesting! The code works with intval() conversions and starts counting from 1000. 
So now that you mention it... this already looks like a big workaround o.O

Comment (jgsf-web) [reply php/examples/users.php:22]: Yes agreed. Maybe the original author was unaware of the difference between login ID and account ID, the latter of which are integers beginning from 1000. These cannot and must not be set or changed.

Comment (SadMonk) [reply php/examples/users.php:22]: Or the original author wanted to keep it as simple as possible and not do any zero-fill string shenanigans in an example implementation.

**Review by jgsf-web — APPROVED** [review]

**Review by jgsf-web — APPROVED** [review]

**Pull request merged**